README.md

```diff
@@ -0,0 +1,47 @@
+# Subspace-SDK
+
+[TODO]: <> (Add docs-rs label (should we generate and host our own one?))
+
+A library for easily running a local Subspace blockchain.
+
+## Dependencies
+
+- rust (check out [`rust-toolchain.toml`](rust-toolchain.toml))
+- protobuf compiler
+- llvm
```

Comment:
I know it seems elementary but you should be more explicit here, something like in [development.md](https://github.com/subspace/subspace/blob/main/docs/development.md) in the monorepo.

Reply:
I suggest you provide guidance on how to install rust via `rustup` and set it to `nightly` to be more explicit. 

Otherwise, more concrete steps on how to install the packages (e.g `apt install clang lldb lld`) could be good here.

Reply:
> I suggest you provide guidance on how to install rust via rustup and set it to nightly to be more explicit.

`cargo` will actually fetch info from `rust-toolchain.toml`. I guess we can suggest just installing __some__ version of rust and cargo so that it will install the toolchain from the file.